pkg/graphgen/parsing.go

```diff
@@ -0,0 +1,178 @@
+package graphgen
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"log"
+	"reflect"
+)
+
+func unionString() string   { return "union" }
```

Comment:
You can define constants for this with 
```
const (
  unionString = "union"
  lexProdString = "lexProduct"
...
)
```
But then you have to remove the brackets in the "calls".